qcodes/tests/dataset/temporary_databases.py

```diff
@@ -25,6 +25,24 @@ def empty_temp_db():
         yield
 
 
```

Comment:
please don't commit large binaries :) `qcodes/tests/dataset/for_timing/db_files/version2/holding_006250_runs.db` (ideally, if you could remove that commit from the git tree...)

Reply:
Sorry, that file got in by mistake. Should be gone now.